### PR TITLE
Sharing an article documentation V1

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1210,17 +1210,40 @@ The details of the article are available on the current page and display all the
...
 When an article has been edited, the details view display the updated date and author besides initial information.
 
 |image164|
 
+
+Sharing a News
+---------------
+
+Once an article is posted, you will be allowed to share it in other spaces. The sharing action is possible from the article details page (and from the article's preview in the activity stream). 
+When you click on the share icon, a popup appears containing a reminder of the title of the article and allowing you to select the distribution space(s), add a description to be displayed in the activity that will be generated and validate the sharing action.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
When you click on the share icon, a popup appears containing a reminder of the title of the article and allowing you to select the destination space(s). You can also add a optional description which will be displayed in the sharing activity.
```

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1210,17 +1210,40 @@ The details of the article are available on the current page and display all the
...
+
+|image165|
+
+ ... note:: Sharing articles is only possible to spaces for the moment, you will be able to select only the spaces in which you are a member.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
 ... note:: Sharing articles is only possible to spaces for the moment. You will be able to select only the spaces in which you are a member.
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1210,17 +1210,40 @@ The details of the article are available on the current page and display all the
...
+ 
+Once the article is shared, a message confirming that the action has been completed is displayed with the list of the destination space(s).
+
+|image166|
+
+
+You can check that the action has been done succesfully by connecting to the concerned spaces. The article preview will be available in the activity stream with the description if this field has been filled in.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
You can check that the action has been done successfully by connecting to the destination spaces. The article preview will be available in the activity stream with the description if this field has been filled in.
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -1151,17 +1151,17 @@ the link.
 .. _Sharing-news:
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: Shouldn't we rename also this ?

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 4 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: I had the same idea but didn't know how to rename it. WDYT of "working-with-news"?

Comment (thomasdelhomenie) [reply docs/GettingStarted.rst]: just "news" ?

Comment (samehaidi) [reply docs/GettingStarted.rst]: OK. I'll do it in the next commit.

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**